trunk/conf/full.conf

```diff
@@ -1758,6 +1758,21 @@ vhost dash.srs.com {
         # Overwrite by env SRS_VHOST_DASH_DASH_MPD_FILE for all vhosts.
         # Default: [app]/[stream].mpd
         dash_mpd_file [app]/[stream].mpd;
+        # The dash windows size in seconds.
+        # Overwrite by env SRS_VHOST_DASH_DASH_WINDOW_SIZE for all vhosts.
+        # Default: 5
+        dash_window_size 5;
+        # whether cleanup the old expired dash files.
+        # Overwrite by env SRS_VHOST_DASH_DASH_CLEANUP for all vhosts.
+        # default: on
+        dash_cleanup on;
+        # If there is no incoming packets, dispose DASH in this timeout in seconds,
+        # which removes all DASH files including m3u8 and ts files.
+        # @remark 0 to disable dispose for publisher.
+        # @remark apply for publisher timeout only, while "etc/init.d/srs stop" always dispose DASH.
+        # Overwrite by env SRS_VHOST_DASH_DASH_DISPOSE for all vhosts.
+        # default: 120
+        dash_dispose 120;
```

Comment:
https://github.com/ossrs/srs/blob/e3d74fb04504cce64f78119627efe3aae825dda3/trunk/src/app/srs_app_config.cpp#L6862-L6866

the default value in the code need to match the value here, be careful, change the default value in the `srs_app_config` may has impact.

Reply:
The previous default value is 0, but hls dispose defaults to 120. I think it's ok to change to the same.

Reply:
I didn't notice the dash_dispose default value in srs_app_config.cpp already changed to 120 to match the full.conf. No problem here.